backend/client-pkg/services/HelloService/index.js

```diff
@@ -0,0 +1,31 @@
+const http = require('http');
+
+const sayHello = (endpoint) => {
+  console.log('Sending hello!');
```

Comment:
Ideally we should have Backend writing to some log file, not doing console.log statements